lib/tasks/multitenant/tenants.rake

```diff
@@ -20,6 +20,13 @@ namespace :multitenant do
       puts(query.any? ? 'Some of the tenants haven\t been suspended' : 'All the right tenants have been suspended')
     end
 
+    desc 'Fix in the background tenant_id missing in alerts, log entries and backend apis'
+    task :fix_missing_in_background => :environment do |_task, relations|
```

Comment:
Does this need `tenant_id` mentioned somewhere in the task name? :thinking:

Reply:
it already has, see description: `multitenant:tenants:fix_missing_in_background[backend_apis,alerts]`

Reply:
You have task description as well, when you do `rake --tasks`

Reply:
If yo uask me to make tenants feature non-confusing, I don't think I can do ;)